.bithoundrc

```diff
@@ -0,0 +1,58 @@
+{
+  "ignore": [
```

Comment:
This seems a very specific list. Is there a reason why, for example, MathJax would not be in extjs.

Also, why thirdparty, third_party, vendor.

Not saying we shouldn't do this just wondering if it is compensating for a lack of discipline in directory organisation. I guess these are all directories whose names are out of our control?